internal/cliwrapper/cliwrapper_test.go

```diff
@@ -52,7 +52,7 @@ func TestPodman_PullImage(t *testing.T) {
 	logger := log.NewTestLogger(t)
 	tests.RemoveImage(logger, tests.TestImageMultiPlatform)
 
-	podman := NewCliWrapper(tests.GetPodmanPath(), logger)
+	podman := NewCliWrapper(tests.GetPodmanPath(), logger, "")
```

Comment:
I think we should try to come up with a test which exercises this with a non-empty third argument.

Reply:
I was kinda curious, so I experimented a bit yesterday and couldn't figure out how to set a `podman` system connection to the local remote socket, but I figured it out this morning:

```bash
$ systemctl --user start podman.socket
$ podman system connection add test unix:///run/user/${UID}/podman/podman.sock
$ podman --connection=test ps
CONTAINER ID  IMAGE                      COMMAND         CREATED       STATUS       PORTS                   NAMES
e3ee2b57514a  localhost/backend:latest                   39 hours ago  Up 39 hours                          back
2e2052ba3580  localhost/frontend:latest  yarn run start  39 hours ago  Up 39 hours  0.0.0.0:3001->3000/tcp  front
```

Interestingly, the global `/run/podman/podman.sock` doesn't work because I "can't read it".

Reply:
I came to the same conclusion last night though since my account was root, was able to access the `/run/podman/podan.sock`. 

Couple caveats I see:
- It sets that as your default connection if you don't have one at all, probably not a big deal. This seems to be the case on my linux test machine. 
- It won't work on macOS but the upside is does have connections already present that can be used:
```
▶  podman system connection list
Name                         URI                                                         Identity                                                       Default     ReadWrite
podman-machine-default       ssh://core@127.0.0.1:53098/run/user/501/podman/podman.sock  /Users/edchong/.local/share/containers/podman/machine/machine  true        true
podman-machine-default-root  ssh://root@127.0.0.1:53098/run/podman/podman.sock           /Users/edchong/.local/share/containers/podman/machine/machine  false       true
```
 
 @webbnh @dbutenhof I was thinking we can add a podman remote test and execute `podman system connection list`, if there is a `podman-machine-default` use that. 
If there isn't one, skip the test. The user will have to set it up a `podman-machine-default` connection if they want to execute the test.

Reply:
Webb pointed out that `podman` can create its own listener; so if the test environment can't run `systemctl --user`, 

```bash
podman system service --time=0 unix:///var/tmp/mypodman.sock &
podman system connection add test unix:///var/tmp/mypodman.sock
podman --connection=test ps
kill %1
```

It doesn't seem like there's a clean way to shut down the "system service" listener; just killing the backgrounded job seems to work OK.